app/views/ships/index.html.erb

```diff
@@ -0,0 +1,7 @@
+<h1>Spaceships</h1>
+<%= link_to 'New Spaceship', new_ship_path %>
+<ul>
+  <% @ships.each do |ship| %>
+    <li><%= ship.name %> - <%= ship.description %></li>
```

Comment:
In the index when showing all ships, we should also add the price per day and the category info of every ship in order to give some more details about the displayed ships, before clicking on one